newsroom/users/views.py

```diff
@@ -37,7 +36,6 @@ def get_view_data():
         'topics': get_user_topics(user['_id']) if user else [],
         'companyName': get_user_company_name(user),
         'locators': get_vocabulary('locators'),
-        'watch_lists': get_watch_lists_for_company(user),
```

Comment:
can you make it somehow configurable? this will create conflicts imo

Reply:
I have added a parameter for now to get CP to prod, I am creating the same parameter change against master to keep it consistent